design/keycloak.x/configuration.md

```diff
@@ -373,6 +373,19 @@ By default only supported features are enabled. In addition there are some suppo
 To enable non-supported features you can either enable individual features or change the feature level with `--features=<level>`. 
 For example `--features=preview` would enable all preview and deprecated features.
 
+### Health and metrics
+
+Enabling/disabling health and metrics information is done with `metrics.enabled=true|false`.
```

Comment:
Could you just add an example of how to enable metrics, something like that should be enough:

```
kc.[sh|bat] config --metrics-enabled=true
```

The reason I'm asking this is that this document is pretty much the best documentation we have at the moment, unfortunately.

Reply:
Forget about it ..